ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/console/GraphConsoleView.java

```diff
@@ -119,6 +146,38 @@ private void createUIComponents() {
         graphCanvas = new JPanel(new GridLayout(0, 1));
         consoleTabsPane = new JBTabsPaneImpl(null, SwingConstants.TOP, this);
         consoleTabs = (JBTabsImpl) consoleTabsPane.getTabs();
+
+        consoleTabs.addTabMouseListener(new MouseListener() {
```

Comment:
There should be MouseAdaptor somewhere, that can be used to override only necessary methods.